templates/cf-jobs.yml

```diff
@@ -694,6 +694,7 @@ properties:
 
   router:
     enable_ssl: (( merge || nil ))
+    enable_routing_api: (( merge || true ))
```

Comment:
I see.  That's unfortunate, hopefully this will be solved once [BOSH links](https://github.com/cloudfoundry/bosh-notes/blob/master/links.md) land.  In the mean time, I'd prefer to avoid increasing the number of places where defaults are set, and would prefer the risk of the CC and router properties' defaults going out of sync.  This doesn't fully solve that problem anyways, in the case where the stub explicitly specifies `enable_routing_api: null`.